bootstrap/kubeadm/main.go

```diff
@@ -122,6 +125,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&watchFilterValue, "watch-filter", "",
 		fmt.Sprintf("Label value that the controller watches to reconcile cluster-api objects. Label key is always %s. If unspecified, the controller watches for all cluster-api objects.", clusterv1.WatchLabel))
 
+	fs.StringVar(&watchFilterExcludedNamespaces, "excluded-namespaces", "",
```

Comment:
Is there a `StringSliceVar`? We are using pflag so I think there is

Reply:
used